controllers/platformnetwork_controller.go

```diff
@@ -335,9 +388,9 @@ func (r *PlatformNetworkReconciler) ReconcileUpdatedAddressPool(client *gophercl
 	return nil
 }
 
-// ReconcileNew is a method which handles reconciling a new data resource and
-// creates the corresponding system resource thru the system API.
-func (r *PlatformNetworkReconciler) ReconciledDeletedAddressPool(client *gophercloud.ServiceClient, instance *starlingxv1.PlatformNetwork, pool *addresspools.AddressPool) error {
```

Comment:
what is the purpose of renaming this API? can we not still use the same API name and have the functional changes ..

Reply:
Hugo suggested to change the name since it's deleting the address to reconcile and not reconciling "deleted" address pool.